packages/types/generate-schemas.ts

```diff
@@ -26,10 +38,13 @@ const HEADER = `/*
  */
 `;
 
+const rootDir = 'src/types';
+const filesToBuildSchemasFor = getTsFiles(rootDir);
+
 const { filename, typesPath } = config as any;
 
 const program = TJS.getProgramFromFiles([resolve(typesPath)]);
-const schemas = TJS.generateSchema(program, '*', settings);
+const schemas = TJS.generateSchema(program, '*', settings, filesToBuildSchemasFor);
```

Comment:
We `import` the `KeysToCamelCase` type into the `src/types/request` module, so it was always getting added to the schema.

Tried a few ways to get the generate schemas to just build against the `src/types` directory, but ultimately had to brute force this.